ext/hook/uhook.stub.php

```diff
@@ -20,7 +21,16 @@ class HookData {
      *                                        provide the proper stack.
      */
     public function span(SpanStack|SpanData|null $parent = null): SpanData;
+
+    /**
+     * Replaces the arguments of a function call. Must be called within a pre-hook.
+     * It is not allowed to pass more arguments to a function that currently on the stack or total number or arguments,
+     * whichever is greater.
+     *
+     * @param array $arguments An array of arguments, which will replace the hooked functions arguments.
+     */
+    public function overrideArguments(array $arguments): void;
```

Comment:
```suggestion
    public function overrideArguments(array $arguments): bool;
```
idea: I was thinking that it would be nice to know if this actually worked so the developer can react on this. I don't know if this would be different from how all the other stuff is currently handled, so it is just a suggestion and non blocking